pages/ksphere/konvoy/1.6.0-beta/addons/dependencies/index.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: layout.pug
+navigationTitle: Dependencies
+title: Addon Dependencies
+menuWeight: 7
+excerpt: Understand addon dependencies and interactions
+enterprise: false
+---
+
+<!-- markdownlint-disable MD018 -->
+
+When deploying Konvoy, addons are a integral part of the cluster's functionality. When deploying or troubleshooting addons, it helps to understand how addons interact and may require other addons as dependencies.
```

Comment:
```suggestion
When deploying Konvoy, addons play an integral part of the clusters functionality. When deploying or troubleshooting addons, it helps to understand how addons interact and may require other addons as dependencies.
```

Reply:
@russurquhart1 this isn't grammatically correct.